src/index.html

```diff
@@ -17,6 +17,50 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img src="./images/logo.png"
+           class="img"
+           alt="moyologo">
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <div src="./images/Rectangle 2.svg"
+               class="rect"
+               alt="line">>
+            <ul class="apple is-active"
+              href="apple.html">APPLE
+            </ul>
+          </div>
+          <ul class="sams"
+              href="samsung.html">SAMSUNG
+          </ul>
+
+          <ul class="smart"
+              href="smartphones.html">SMARTPHONES
+          </ul>
+
+          <ul class="lapcom"
+              data-qa="hover"
+              href="laptops&computers.html">LAPTOPS & COMPUTERS
+          </ul>
```

Comment:
The `<ul>` element should be replaced with an `<li>` element for navigation purposes, and the text should be wrapped in an `<a>` tag.